### ISSUE TITLE
Update variable on every panel

### ISSUE THREAD

**Issue by stefi994** [grafana/grafana#30078]
**What would you like to be added**:

Hi guys, not sure if this is already added so please correct me. I have two variables $Interface and $Description. I am doing a repeat panel by $Interface and this is working well, unfortunately the $Description is not updated on every panel as the $Interface.

Do I need to combine the variables somehow so both are updated on every panel? Do I need to change something?

Basically I need the second variable $Description to be updated on every panel as well.

**Why is this needed**:
I need this so that a have the correct panel title:
eth0 desc0
eth1 desc1
...

Instead of the current:
eth0 desc0
eth1 desc0
...

**Comment by torkelo** [comment]
You cannot repeat based on two nested variables. the $Desription variable will be the same for every repeated panel as you can only repeat based on one variable

**Comment by stefi994** [comment]
Okay, is there maybe a way to combine a result of query which is text, i.e. the $Description, and that to write it in the Panel Title or include in the Legend?
